dali/python/nvidia/dali/backend.py

```diff
@@ -46,10 +46,10 @@ def deprecation_warning(what):
         deprecation_warning("DALI support for Python 3.10 is experimental and some functionalities "
                             "may not work.")
 
-    if __cuda_version__ < 102:
-        deprecation_warning("DALI 1.3 is the last official release that supports CUDA 10.0. "
-                            "The next release will support only 10.2 from 10.x familly. "
-                            "Please update your environment to CUDA version 10.2 or newer.")
```

Comment:
This was a dead code as the `__cuda_version__` is build time thingy and we don't build with cuda 10.0.